docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: '3'
+
+services:
+  client:
+    build:
+      context: .
+      dockerfile: Dockerfile.client
+    image: elifesciences/peerscout_client:${IMAGE_TAG}
+    command: /bin/sh -c exit 0
+
+  peerscout:
+    build:
+      context: .
+      dockerfile: Dockerfile
+      args: 
+        commit: ${IMAGE_TAG}
+    image: elifesciences/peerscout:${IMAGE_TAG}
+    command: /bin/sh -c exit 0
+    depends_on:
+      - client
+
+  peerscout-base-dev:
+    build:
+      context: .
+      dockerfile: Dockerfile
+      args: 
+        commit: ${IMAGE_TAG}
+        install_dev: y
+    image: elifesciences/peerscout_dev_base:${IMAGE_TAG}
```

Comment:
image name inconsistent with service name?

Reply:
You mean it should use hyphen? I only used underscore here because we previously used the `_ci` suffix.

Reply:
just the order of the words `dev`/`base`

Reply:
(underscores are fine as they are truly separating different parts of the image name rather than words inside a project name, e.g. `sciencebeam-judge_ci`)

Reply:
Hmm I was actually considering hyphen because it would match the service name